cookbooks/tian_parameterization_kinematic_slab/coupled_two_phase_tian_parameterization_kinematic_slab.prm

```diff
@@ -0,0 +1,234 @@
+# This is a cookbook which simulates a portion of 2D subduction system with
+# a hydrated sediment layer, MORB layer, gabbro layer, and peridotite
+# mantle wedge. The layers dehydrate as they are subducted, with the solid-fluid
+# reactions governed by an approximation published by Tian et al., 2019.
+# The model simulates the upper 20 km of a slab subducting with a dip of 45 degrees
+# and a convergence rate of 5 cm/yr. The Cartesian box is rotated such that the
+# x-axis aligns with the slab surface, and the y-axis is orthogonal to the slab
+# surface, while gravity is imposed at an angle of 45 degrees. A surface pressure
+# of 1 GPa is imposed to account for the model simulating a portion of the slab
+# that is already subducted.
+# For stability, the slab first subducts for 100 kyr before the hydration
+# is added, and the slab surface temperature diffuses to the mantle over 10 km
+# perpendicular to the slab surface. From slab surface to the base of the model,
+# the composition is comprised of a 5km thick layer of sediment with 3 wt% water,
+# below is a 7km thick layer of MORB with 2 wt% water, and below that is an
+# 8 km thick layer of gabbro with 1 wt% water.
+
+set Adiabatic surface temperature              = 1600
+set Nonlinear solver scheme                    = iterated Advection and Stokes
```

Comment:
It would be good to try this with the Newton solver to see how the performance changes for both the current and stricter nonlinear solver tolerances.

Reply:
Just keep in mind that we have not implemented any derivatives for the melt terms (so we do not have the Jacobian for the Newton solver--not sure if it has ways around that now).